pkgs/tools/admin/salt/fix-libcrypto-loading.patch

```diff
@@ -1,13 +1,13 @@
-diff --git a/salt/utils/rsax931.py b/salt/utils/rsax931.py
-index f827cc6db8..b728595186 100644
---- a/salt/utils/rsax931.py
-+++ b/salt/utils/rsax931.py
-@@ -74,7 +74,7 @@
+--- a/salt/utils/rsax931.py	2021-11-24 00:39:57.940790184 +0100
++++ b/salt/utils/rsax931.py	2021-11-24 00:38:35.436728341 +0100
+@@ -85,6 +85,10 @@
      """
      Attempt to load libcrypto.
      """
--    return cdll.LoadLibrary(_find_libcrypto())
-+    return cdll.LoadLibrary('@libcrypto@')
++    try:
++        return cdll.LoadLibrary('@libcrypto@')
++    except OSError:
++        pass
+     return cdll.LoadLibrary(_find_libcrypto())
```

Comment:
Why not?
```suggestion
+    try:
+        return cdll.LoadLibrary('@libcrypto@')
+    except OSError:
+        return cdll.LoadLibrary(_find_libcrypto())
```

Reply:
Feel free to change it. There may be many permutations of the code that do the same. I like the current version because it makes explicit we are trying a non-default location and ignoring if it does not work. The usual location stays indented at the top level.